asset/asset.go

```diff
@@ -92,6 +92,30 @@ const (
 	V1 Version = 1
 )
 
+// GroupKeyVersion denotes the version of the group key construction.
+type GroupKeyVersion uint8
```

Comment:
Why isn't the version we added to just the group key reveal sufficient?

Reply:
I think you're right. I think we can re-use group key reveal version type here. Will update.